fixtures/lang-frontends/php-laravel/app/Jobs/SomeJob.php

```
@@ -0,0 +1,7 @@
+<?php
+
+namespace App\Jobs;
+
+class SomeJob implements ShouldQueue
```

PHPStan (2.2.7)

[error] Class App\Jobs\SomeJob implements unknown interface App\Jobs\ShouldQueue.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(interface.notFound)